lib/parsers/gjs-gts-parser.js

```diff
@@ -291,6 +291,16 @@ function preprocessGlimmerTemplates(info, code) {
       n.type = `Glimmer${n.type}`;
       allNodeTypes.add(n.type);
     }
+    // ast should not contain comment nodes
+    for (const comment of comments) {
+      const parentBody = comment.parent.body || comment.parent.children;
+      const idx = parentBody.indexOf(comment);
+      parentBody.splice(idx, 1);
+      // comment type can be a block comment or a line comment
+      // block comments start with <!--- or  {{!--, line comments with <!-- or {{!
```

Comment:
{{!}} Are also block comments

Reply:
how so? `{{!` only comments out 1 line.
i just need to distinguish it because eslint handles those differently.
html comments has issues... https://github.com/glimmerjs/glimmer-vm/issues/1487

Reply:
Demo: https://limber.glimdown.com/edit?c=MQAgwg9gtlCmB2AXAzgKFQAywcwFbJABsBLAN1lQB5FYoAHQgQxoD5UQQAJWQwiAGhABxEjFgAnAIToOAb1nSOHKAFdCiYuyUl4sEFoC%2BBmSHmSAtOa3K1G60WK6tloycoXz%2BpSFXrN3nQoOSzYtd0tPe18NBydgkKoAehp6JlZMLHQgA&format=glimdown

Reply:
> html comments has issues

Yeah, i saw that. Mourd be good to fix. 
I'm almost done with getting the upgrade in ember figured out

Reply:
btw,
```html
<!--- 
    multi line
  --->
```
did not make it into the output